### PR TITLE
Fix token dependent error messages.

### LINKED ISSUE #11: Token Expiration Error
We should display an error that the token is expired and ask the user to generate again.

### LINKED ISSUE #13: Token File Missing Message
Change the error message when token file isn’t present in specified directory.

The message should be:
```
The authentication token json file doesn't exists at the required path.
Please download the file from the Profile section of the EvalAI webapp and place it at ~/.evalai/token.json
```

### PR DESCRIPTION
**Pull request by isht3**
Fixes #13 
Fixes #11

### CODE REVIEW

**Comment by isht3** [comment]
@RishabhJain2018 Done!

**Comment by isht3** [comment]
@RishabhJain2018 Updated the tests too!

File: tests/test_auth.py [hunk 1]
```diff
@@ -15,8 +19,47 @@ def setup(self):
...
+    def setup(self):
+
+        token_expired_data = json.loads(challenge_response.token_expired)
+
+        url = "{}{}"
+        responses.add(responses.GET, url.format(API_HOST_URL, URLS.challenge_list.value),
+                      json=token_expired_data, status=401)
+
+    @responses.activate
+    def test_diaplay_all_challenge_lists_when_token_has_expired(self):
```
Comment (RishabhJain2018) [inline tests/test_auth.py]: Ahh..one typo `test_display_all_challenge_lists_when_token_has_expired`

File: tests/test_auth.py [hunk 2]
```diff
@@ -15,8 +19,47 @@ def setup(self):
...
+        expected = ("\nThe authentication token json file doesn't exists at the required path. "
+                    "Please download the file from the Profile section of the EvalAI webapp and "
+                    "place it at ~/.evalai/token.json\n\n")
+        runner = CliRunner()
+        result = runner.invoke(challenges)
+        response = result.output
+        assert response == expected
+
+
+class TestRequestWithInvalidToken(BaseTestClass):
```
Comment (RishabhJain2018) [inline tests/test_auth.py]: Change this too then 😄 
`TestUserRequestWithInvalidToken`

File: tests/test_auth.py [hunk 3]
```diff
@@ -15,8 +19,47 @@ def setup(self):
...
+    @responses.activate
+    def test_diaplay_all_challenge_lists_when_token_is_invalid(self):
+        expected = "\nThe authentication token you are using isn't valid. Please generate it again.\n\n"
+        runner = CliRunner()
+        result = runner.invoke(challenges)
+        response = result.output
+        assert response == expected
+
+
+class TestRequestWithExpiredToken(BaseTestClass):
```
Comment (RishabhJain2018) [inline tests/test_auth.py]: `TestUserRequestWithExpiredToken`

**Pull request merged**